### ISSUE TITLE
AMC no longer working

### ISSUE THREAD

**Issue by Conroman16** [filebot/filebot#15]
Hey there,

Just today I've started having an issue with FileBot's AMC script.  Yesterday things were working fine but today I'm getting this error in my logs

```
Script not found: amc
java.io.FileNotFoundException: Script not found: amc
	at net.filebot.cli.ScriptBundle.getScript(ScriptBundle.java:54)
	at net.filebot.cli.ScriptShell.runScript(ScriptShell.java:72)
	at net.filebot.cli.ArgumentProcessor.runScript(ArgumentProcessor.java:113)
	at net.filebot.cli.ArgumentProcessor.run(ArgumentProcessor.java:28)
	at net.filebot.Main.main(Main.java:124)
Failure (°_°)
```

On the surface it looks like the AMC script isn't being found, however I've since cloned the filebot scripts repo and manually directed filebot at the `amc.groovy` file and I'm still having the same issue.

As for my system, nothing has changed whatsoever between yesterday when it was working and today when it was not.  Is there any support you can lend?

Any input would be much appreciated as that AMC script is the backbone of my media server and I'd hate to have to tear it out and replace it with a sonarr/radarr-based system over this issue

EDIT:  If you need any more logs or any other info just let me know!  Thanks so much

**Comment by rednoah** [comment]
What does `filebot -version` say?

**Comment by Conroman16** [comment]
The output is `FileBot 4.7.8 (r4846) / Java(TM) SE Runtime Environment 1.8.0_121 / Windows Server 2012 R2 6.3 (amd64)`.

**Comment by rednoah** [comment]
How about `filebot -script fn:sysinfo`?

**Comment by Conroman16** [comment]
Sure thing.  The output from that is
```
FileBot 4.7.8 (r4846)
JNA Native: 5.1.0
MediaInfo: 0.7.92.1
7-Zip-JBinding: 9.20
Chromaprint: 1.4.2
Extended Attributes: OK
Unicode Filesystem: OK
Script Bundle: 2017-03-24 (r490)
Groovy: 2.4.8
JRE: Java(TM) SE Runtime Environment 1.8.0_121
JVM: 64-bit Java HotSpot(TM) 64-Bit Server VM
CPU/MEM: 6 Core / 910 MB Max Memory / 22 MB Used Memory
OS: Windows Server 2012 R2 (amd64)
Package: MSI
Data: C:\Users\connor\AppData\Roaming\FileBot
uname: MSYS_NT-6.3 tbox 2.5.0(0.295/5/3) 2016-03-31 18:47 x86_64 Msys
Done ?(?????)?
```

Yep, I'm fairly sure it's the same version.  I'm calling it from my PATH in both cases.  I didn't see that thread over there on the forums yet.  It's interesting that their AMC script disappeared at nearly the same time as mine did.

**Comment by rednoah** [comment]
If the `sysinfo` script works, then the `amc` script must work as well. Try `filebot -script fn:amc`.

**Comment by rednoah** [comment]
So I've looked into this now.

This stacktrace could not have been produced by `FileBot 4.7.8 (r4846)`:
```
Script not found: amc
java.io.FileNotFoundException: Script not found: amc
	at net.filebot.cli.ScriptBundle.getScript(ScriptBundle.java:54)
	at net.filebot.cli.ScriptShell.runScript(ScriptShell.java:72)
	at net.filebot.cli.ArgumentProcessor.runScript(ArgumentProcessor.java:113)
	at net.filebot.cli.ArgumentProcessor.run(ArgumentProcessor.java:28)
	at net.filebot.Main.main(Main.java:124)
Failure (°_°)
```

Proof:
```
FileBot 4.7.8 (r4846) / Java(TM) SE Runtime Environment 1.8.0_121 / Mac OS X 10.12.3 (x86_64)
Script not found: test
java.io.FileNotFoundException: Script not found: test
	at net.filebot.cli.ScriptBundle.getScript(ScriptBundle.java:55)
	at net.filebot.cli.ScriptShell.runScript(ScriptShell.java:74)
	at net.filebot.cli.ArgumentProcessor.runScript(ArgumentProcessor.java:122)
	at net.filebot.cli.ArgumentProcessor.run(ArgumentProcessor.java:29)
	at net.filebot.Main.main(Main.java:115)
```
Note that the line numbers don't match the line numbers in your stack trace, so we know it's a different build.

**Comment by Conroman16** [comment]
And the plot thickens!  Good catch.  I'll have to dig into this a bit and get back to you.  I'm AFK for a while but should have something for you within the next day

**Comment by Conroman16** [comment]
So it ended up being some windows PATH fuckery.  Basically when I installed filebot the very first time, I installed it as "Administrator" but then the next time I installed it as my user.

Then somewhere along the lines, I changed the user the daemon runs under and at that point, it was calling  the outdated version of filebot in the system PATH and not the up-to-date version in my user's PATH.

I went in and cleaned everything out and then reinstalled the most recent version.  Now things are working again!  Had you not pointed out that the lines in the stack trace were different, I would have believed there was only one version installed.  Thanks so much for your help!  I just sent you a donation on PayPal for the support and as a thank you for not making me use the windows store haha.  Cheers!
